app/src/test/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntryEffectHandlerTest.kt

```diff
@@ -110,4 +114,20 @@ class BloodSugarEntryEffectHandlerTest {
     verify(ui).showDateOnDateButton(bloodSugarDate)
     verifyNoMoreInteractions(ui)
   }
+
+  @Test
+  fun `set date on input fields and date button when prefill date effect is received`() {
+    // given
+    val entryDate = LocalDate.of(1992, 6, 7)
+    userClock.setDate(LocalDate.of(1992, 6, 7), ZoneOffset.UTC)
```

Comment:
We can skip the second parameter to set the timezone since `TestUserClock` already has a default timezone of `UTC`.